tests/integration/docusaurus/connecting_to_your_data/connect_to_your_data_overview.py

```diff
@@ -0,0 +1,34 @@
+import pathlib
+import great_expectations as gx
+
+data_directory = str(
+    pathlib.Path(
+        gx.__file__,
+        "..",
+        "..",
+        "tests",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+    ).resolve(strict=True)
+)
```

Comment:
```suggestion
data_directory = pathlib.Path(
        gx.__file__,
        "..",
        "..",
        "tests",
        "test_sets",
        "taxi_yellow_tripdata_samples",
    ).resolve(strict=True)
```
This doesn't need to be an `str` anymore.